app/scripts/components/root-discoveries/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { discoveries } from 'veda/thematics';
+
+import { LayoutProps } from '$components/common/layout-root';
+import PageHero from '$components/common/page-hero';
+import { Fold, FoldHeader, FoldTitle } from '$components/common/fold';
+import { Card, CardList } from '$components/common/card';
+import EmptyHub from '$components/common/empty-hub';
+
+import { PageMainContent } from '$styles/page';
+import { thematicDiscoveriesPath } from '$utils/routes';
+
+/* eslint-disable-next-line fp/no-mutating-methods */
+const allDiscoveries = Object.values(discoveries)
+  .map((d) => d!.data)
+  .sort((a, b) => {
+    const getTime = (d: string) => {
+      const millis = new Date(d).getTime();
+      return isNaN(millis) ? -Infinity : millis;
+    };
+
+    return getTime(b.pubDate) - getTime(a.pubDate);
```

Comment:
What is the reason for sorting by pub date rather than discovery title? Shouldn't this be coherent with dataset sorting?

Reply:
Since the discoveries are more akin to news articles, I think that novelty is important, so the most recent ones are shown first.
Datasets are more of exploratory nature so they're easier to find if alphabetical.

Reply:
That's a sound rationale but in that case, the design should probably be different. Anyways, something for later.